docs/develop/parent-to-dialog.md

```diff
@@ -1,100 +1,49 @@
 ---
-title: Passing data and messages to a dialog box from its host page
-description: 'Learn how to pass data to a dialog from the host page by using the messageChild and DialogParentMessageReceived APIs.'
-ms.date: 07/07/2020
+title: Alternative ways of passing messages to a dialog box from its host page
+description: 'Learn workarounds to use when the messageChild method is not supported.'
+ms.date: 08/20/2020
+
 localization_priority: Normal
 ---
 
-# Passing data and messages to a dialog box from its host page (preview)
-
-Your add-in can send messages from the [host page](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page) to a dialog box using the [messageChild](/javascript/api/office/office.dialog#messagechild-message-) method of the [Dialog](/javascript/api/office/office.dialog) object.
+# Alternative ways of passing messages to a dialog box from its host page
 
-> [!Important]
->
-> - The APIs described in this article are in preview. They are available to developers for experimentation; but should not be used in a production add-in. Until this API is released, use the techniques described in [Pass information to the dialog box](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box) for production add-ins.
-> - The APIs described in this article require a Microsoft 365 subscription. You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://insider.office.com). Please note that when a build graduates to the production semi-annual channel, support for preview features is turned off for that build.
-> - In the initial stage of the preview, the APIs are supported in Excel, PowerPoint, and Word; but not in Outlook.
->
-> [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+The recommended way to pass data and messages from a parent page to a child dialog box is with the `messageChild` method as described in [Use the Office dialog API in your Office Add-ins](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box). But if your add-in is running on a platform or host that does not support the [DialogApi 1.2 requirement set](../reference/requirement-sets/dialog-api-requirement-sets.md), there are two other ways that you can use to pass information to the dialog box:
```

Comment:
Nit: removing the sentence-starting "but". Also shortening the last sentence a bit.
```suggestion
The recommended way to pass data and messages from a parent page to a child dialog box is with the `messageChild` method as described in [Use the Office dialog API in your Office Add-ins](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box). If your add-in is running on a platform or host that does not support the [DialogApi 1.2 requirement set](../reference/requirement-sets/dialog-api-requirement-sets.md), there are two other ways that you can pass information to the dialog box:
```